pages/bare_metal_cloud/dedicated_servers/rcw-changing-admin-password-on-windows/guide.en-gb.md

```diff
@@ -0,0 +1,169 @@
+---
+title: "Reset Administrator password when using Rescue-Customer-Windows"
+excerpt: "Reset Administrator password when using Rescue-Customer-Windows"
+updated: 2024-05-21
+---
+
+## Objective
+
+This guide will help you to reset your `Administrator`'s password with the new __*rescue-customer-windows*__
```

Comment:
```suggestion
This guide will help you to reset your `Administrator`'s password with the __*Windows customer rescue system*__
```
I dropped the "new" because it generally doesn't age well. We've set requirements forcing users to either see the "old" WinPE rescue or the "new" one so I don't think we should bother about "new" here.